torch/_dynamo/output_graph.py

```diff
@@ -597,7 +597,7 @@ def get_submodule(self, keys):
     def new_var(self, name="tmp"):
         existing = set(self.code_options["co_varnames"])
         for i in itertools.count():
-            var = f"___{name}_{i}"
+            var = f"{name}_{i}"
```

Comment:
Any particular reason for this now? No, right?

Reply:
No, I just never liked how these looked